apps/www/components/Article/ReadAloudInline.tsx

```diff
@@ -59,21 +59,37 @@ const styles = {
     margin: 0,
   }),
   title: css({
-    ...fontStyles.sansSerifMedium15,
+    ...fontStyles.sansSerifMedium16,
     textDecoration: 'none',
     cursor: 'pointer',
-    margin: '0px 0 4px 0',
     padding: 0,
   }),
+  link: css({
+    ...fontStyles.sansSerif16,
+  }),
 }
 
-const SyntheticAudio = ({ meta, t }: { meta: Meta; t: (sting) => string }) => {
+const ReadAloudInline = ({ meta, t }: { meta: Meta; t: (sting) => string }) => {
   const { toggleAudioPlayer } = useContext<AudioContextType>(AudioContext)
   const [colorScheme] = useColorContext()
   const { kind } = meta.audioSource
   const isSynthetic = kind === 'syntheticReadAloud'
-  const Icon = isSynthetic ? AudioIcon : PodcastIcon
-  const eventCategory = isSynthetic ? 'SyntheticAudio' : 'ReadAloudAudio'
+  const isReadAloud = kind === 'readAloud'
+
+  const { Icon, eventCategory, title, label, href } = useMemo(
+    () => ({
+      Icon: (isSynthetic && AudioIcon) || PodcastIcon,
+      eventCategory: (isSynthetic && 'SyntheticAudio') || 'ReadAloudAudio',
+      title: t(`article/${kind}/title`),
+      label: t(`article/${kind}/link`),
+      href:
+        (isSynthetic && '/synthetic-read-aloud') ||
+        (isReadAloud && '/read-aloud') ||
+        false,
+    }),
+    [kind],
+  )
```

Comment:
The `useMemo` is more expensive than it's worth for simple logic checks, also just depending on `kind`would not suffice (at least from a linters perspective)
```suggestion
  const Icon = (isSynthetic && AudioIcon) || PodcastIcon
  const eventCategory = (isSynthetic && 'SyntheticAudio') || 'ReadAloudAudio'
  const title = t(`article/${kind}/title`)
  const label = t(`article/${kind}/link`)
  const href = (isSynthetic && '/synthetic-read-aloud') || (isReadAloud && '/read-aloud') || false
```